src/checks.py

```diff
@@ -215,7 +216,11 @@ def check_no_binary_files(state: State) -> Optional[str]:
 
 
 def check_build_and_test(state: State) -> Optional[str]:
-    return _check_sh("./mvnw --quiet package", workdir=state.source_dir)
+    return _check_sh(
```

Comment:
I'd prefer having two separate commands - `_check_sh` can take a list of commands as its first argument, and it'll bail out on the first non-zero exit code (so `_check_sh(["mvn ...", "./mvnw ..."])`). This gives us cleaner output.